bin/submit-task

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## This script submits new tasks for SAPS interacting with Dispatcher backend.
+
+## Checking args
+if [ $# -ne 12 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD LOWER_LEFT_LATITUDE UPPER_RIGHT_LATITUDE LOWER_LEFT_LONGITUDE UPPER_RIGHT_LONGITUDE PROCESSING_INIT_DATE PROCESSING_FINAL_DATE INPUTDOWNLOADING_TAG PREPROCESSING_TAG PROCESSING_TAG DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+## args
+USER_EMAIL=$1
+USER_PASSWORD=$2
+LOWER_LEFT_LATITUDE=$3
+UPPER_RIGHT_LATITUDE=$4
+LOWER_LEFT_LONGITUDE=$5
+UPPER_RIGHT_LONGITUDE=$6
+PROCESSING_INIT_DATE=$7
+PROCESSING_FINAL_DATE=$8
+INPUTDOWNLOADING_TAG=$9
+PREPROCESSING_TAG=${10}
+PROCESSING_TAG=${11}
+DISPATCHER_ADDRESS=${12}
+
+## constants
+TASK_PRIORITY=0
+
+curl --request POST "http://$DISPATCHER_ADDRESS/processings" \
+  --header "Content-Type: text/plain"  \
+  --data-raw "userEmail=$USER_EMAIL&userPass=$USER_PASSWORD&lowerLeft[]=$LOWER_LEFT_LATITUDE&lowerLeft[]=$LOWER_LEFT_LONGITUDE&upperRight[]=$UPPER_RIGHT_LATITUDE&upperRight[]=$UPPER_RIGHT_LONGITUDE&initialDate=$PROCESSING_INIT_DATE&finalDate=$PROCESSING_FINAL_DATE&inputGatheringTag=$INPUTDOWNLOADING_TAG&inputPreprocessingTag=$PREPROCESSING_TAG&algorithmExecutionTag=$PROCESSING_TAG&email=$USER_EMAIL&priority=$TASK_PRIORITY"
```

Comment:
Add braces on variables for disambiguation. e.g. `${DISPATCHER_ADDRESS}`.

Reply:
Done

Reply:
Do it for all variables in curl request.

Reply:
Done